src/constants/index.js

```diff
@@ -32,16 +32,17 @@ SUPPORTED_LANGUAGES.forEach(lang => {
   }
 });
 
+const public_url = process.env.PUBLIC_URL;
```

Comment:
This seems super small, but see if you can make constants upper-case - it's a common programming habit!